gravitybridge/chain.json

```diff
@@ -157,6 +165,10 @@
       {
         "address": "https://gravitybridge-grpc.lavenderfive.com/",
         "provider": "Lavender.Five Nodes 🐝"
+      },
+      {
+        "address": "https://grpc.gravity.bh.rocks:443/",
+        "provider": "Lavender.Five Nodes 🐝"
```

Comment:
Guessing you forgot to change the provider here?

Reply:
yes, sorry about that...